internal/exec/exec.go

```diff
@@ -0,0 +1,749 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package exec is a fork of the go stdlib os/exec package (and other related packages it calls into)
+// that exposes some additional functionality, namely the ability to use the STARTUPINFOEXW struct
+// for the win32 API CreateProcess function.
```

Comment:
In this comment or maybe in the commit message we should include the commit ID of the go os/exec project from which this is forked and we should try to keep the file names and the directory structure of this code as identical as possible to the upstream code so that in the future if we need to update this fork / remove it completely it will be easier.

Reply:
The filenames and dir structure are already wrong :). The code in process.go here is from exec.go in the os package. The os/exec and os packages are tightly intertwined and there was quite a lot of unused things /things imported from /internal/syscall/windows in the stdlib (which obviously we can't access) that I just scrapped to get the thing to build. I'd done this so long ago I don't even remember what they were. 

What's currently here is just what we need from os and os/exec for their friendly stdlib wrapper of "run a process" to work.   I see the merit for keeping it as identical as possible though. I guess we could have internal/os and then internal/os/exec to keep the structure the same, and then the n files we care about. I don't envision this changing too much though. I was thinking we'd just add what we need to this moving forward but if it looks like we're having to (or wanting to) pull in functionality from the stdlib often (or just once more) maybe we just write our own at that point. This one addition seemed like the only sore spot we really needed that they didn't let us configure, and it doesn't sound like they're keen on adding it either 😢